app/site/controllers.py

```diff
@@ -127,7 +127,7 @@ def search_package():
     q = request.args.get('q')
     if q is None:
         q = ''
-    datapackage_list = DataPackageQuery(query_string=q.strip()).get_data(20)
+    datapackage_list = DataPackageQuery(query_string=q.strip()).get_data(1000)
```

Comment:
`limit=1000`

Would be clearer code. Also why is `limit` not an argument to the constructor (vs an argument to get_data)